runtime/jvmti/jvmtiHelpers.c

```diff
@@ -1676,14 +1676,27 @@ findDecompileInfoFrameIterator(J9VMThread *currentThread, J9StackWalkState *walk
 }
 
 UDATA
-findDecompileInfo(J9VMThread *currentThread, J9VMThread *targetThread, UDATA depth, J9StackWalkState *walkState)
+findDecompileInfo(
+		J9VMThread *currentThread, J9VMThread *targetThread,
+#if JAVA_SPEC_VERSION >= 19
+		j9object_t continuationObject,
+#endif /* JAVA_SPEC_VERSION >= 19 */
+		UDATA depth, J9StackWalkState *walkState)
 {
-	walkState->walkThread = targetThread;
 	walkState->flags = J9_STACKWALK_ITERATE_FRAMES | J9_STACKWALK_INCLUDE_NATIVES | J9_STACKWALK_VISIBLE_ONLY | J9_STACKWALK_RECORD_BYTECODE_PC_OFFSET | J9_STACKWALK_MAINTAIN_REGISTER_MAP;
 	walkState->skipCount = depth;
 	walkState->userData1 = (void *)JVMTI_ERROR_NO_MORE_FRAMES;
 	walkState->frameWalkFunction = findDecompileInfoFrameIterator;
-	currentThread->javaVM->walkStackFrames(currentThread, walkState);
+	if (NULL != targetThread) {
+		walkState->walkThread = targetThread;
+		currentThread->javaVM->walkStackFrames(currentThread, walkState);
+	}
+#if JAVA_SPEC_VERSION >= 19
+	else if (NULL != continuationObject) {
+		J9JavaVM *vm = currentThread->javaVM;
+		vm->internalVMFunctions->walkContinuationStackFrames(currentThread, continuationObject, walkState);
+	}
```

Comment:
if would prefer to have it
```c++
#ifdef 19
if (NULL != continuation) {
    ...
} else
#endif
{
    assert (NULL != targetThread)
    ...
}
```
This ensure that either a continuation or targetThread is supplied